src/racer/core.rs

```diff
@@ -796,6 +800,18 @@ impl<'c> Session<'c> {
         raw.contains_key(path) && masked.contains_key(path)
     }
 
+    /// Find the library root for kratename
+    ///
+    /// The library is searched for by checking
+    ///
+    /// 1. overrides
+    /// 2. lock file
+    /// 3. toml file
+    pub fn get_crate_file(&self, kratename: &str, from_path: &path::Path) -> Option<path::PathBuf> {
```

Comment:
What do you think about making this `pub(crate)` instead? With the 2.0 release, we dramatically reduced the public API surface. At this point, I'd like to take a conservative approach with expanding that API at all.

Reply:
Sure, still have to get used to `pub(crate)`.

Reply:
This is now done.